src/main/java/nl/aerius/sldgenerator/generator/SldUtils.java

```diff
@@ -108,7 +108,7 @@ private static void appendRulePartToSLD(final StringBuilder sld, final SldRule s
 
   private static String getRuleTitle(final SldRule sldRule, final ZoomLevel zoomLevel) {
     final String ruleTitle;
-    if (StringUtils.isEmpty(sldRule.getCustomConditionSld())) {
+    if (StringUtils.isEmpty(sldRule.getCustomConditionSldAsString())) {
```

Comment:
Probably better to do these kind of checks on the `List` instead of the `String` as it will invoke `String,join()` multiple times unnecessarily.

Reply:
Moved the result of the method to a variable instead (because checking on a list is not nice `list == null || list.isEmpty()`)